provider/ansible/documentation.rb

```diff
@@ -20,6 +20,7 @@
 module Provider
   module Ansible
     # Responsible for building out YAML documentation blocks.
+    # rubocop:disable Metrics/ModuleLength
```

Comment:
Consider refactoring your code rather than disabling rubocop. 

however, because it would make things easier to me if the rubocop run was green, I am fine if you commit this but please create an issue to track the refactoring and add TODOs linking back to that issue in the code.